src/tools/rbd/action/Import.cc

```diff
@@ -452,6 +452,11 @@ int do_import_diff(librados::Rados &rados, librbd::Image &image,
   if (strcmp(path, "-") == 0) {
     fd = STDIN_FILENO;
   } else {
+    if (utils::is_blk_dev(path)) {
+      cerr << "rbd: importing diffs from raw block devices "
+           << "isn't currently supported." << std::endl;
```

Comment:
The ``import-diff`` code path will fail with Windows block devices, for example this ``fstat`` call: https://github.com/ceph/ceph/blob/c870ead3d44fe5a52b18f8914f9cbc82bc50ea15/src/tools/rbd/action/Import.cc#L389

It's going to say that ``rbd: failed to stat specified diff file`` instead of the fact that this operation is not supported. We can either add a platform check and only do this on Windows, or just remove these checks altogether. Importing diffs from a block device would be very unusual anyway.